packages/teleport/src/DesktopSession/DesktopSession.tsx

```diff
@@ -124,16 +126,18 @@ export function DesktopSession(props: State) {
           flex: 1, // ensures the canvas fills available screen space
         }}
         tdpCli={tdpClient}
-        tdpCliOnImageFragment={onImageFragment}
+        tdpCliOnPngFrame={onPngFrame}
         tdpCliOnTdpError={onTdpError}
         tdpCliOnWsClose={onWsClose}
         tdpCliOnWsOpen={onWsOpen}
+        tdpCliOnClientScreenSpec={onClientScreenSpec}
         onKeyDown={onKeyDown}
```

Comment:
We can save space and clean this up a little by replacing line 134-140 with `{...props}` here and not destructuring them on line 47-53

Reply:
That would make it more difficult to read imo.